### PR TITLE
[TCDR-2137] Enclone deprecated

### PR DESCRIPTION
**Pull request by abhijna**
TCDR-2137

### CODE REVIEW

File: pages/index.html.src [hunk 1]
```diff
@@ -7,8 +7,15 @@
 <img src="img/enclone_banner.png" alt="enclone banner" title="enclone banner" width=100% />
 </a>
 
+<p style="border: 2px; border-style: solid; border-color: black; background-color: #EBF4FC; 
+padding: 8px; font-size: 110%">
+#The enclone tool is deprecated and will no longer receive updates or feature enhancements.
```
Comment (abhijna) [inline pages/index.html.src:12]:
```suggestion
#enclone is deprecated and will no longer receive updates or feature enhancements. 
```

File: pages/index.html.src [hunk 2]
```diff
@@ -7,8 +7,15 @@
 <img src="img/enclone_banner.png" alt="enclone banner" title="enclone banner" width=100% />
 </a>
 
+<p style="border: 2px; border-style: solid; border-color: black; background-color: #EBF4FC; 
+padding: 8px; font-size: 110%">
+#enclone is deprecated and will no longer receive updates or feature enhancements. 
+#It will remain available for a limited time but will be fully disabled the future. We recommend transitioning to the enhanced functionality now integrated directly within Cell Ranger.
```
Comment (abhijna) [inline pages/index.html.src:13]:
```suggestion
#It will remain available for a limited time but will be disabled the future. We recommend transitioning to the enhanced functionality now integrated directly within Cell Ranger.
```

File: pages/index.html.src [hunk 3]
```diff
@@ -7,8 +7,15 @@
 <img src="img/enclone_banner.png" alt="enclone banner" title="enclone banner" width=100% />
 </a>
 
+<p style="border: 2px; border-style: solid; border-color: black; background-color: #EBF4FC; 
+padding: 8px; font-size: 110%">
+#enclone is deprecated and will no longer receive updates or feature enhancements. 
+#It will remain available for a limited time but will be disabled the future. We recommend transitioning to the enhanced functionality now integrated directly within Cell Ranger.
```
Comment (abhijna) [inline pages/index.html.src:13]:
```suggestion
#It will remain available for a limited time but will be disabled the future. We recommend transitioning to the improved functionality now integrated directly within Cell Ranger.
```

File: index.html [hunk 4]
```diff
@@ -239,7 +239,7 @@ <h2>Inputs to <span style="color:rgb(120,123,175);font-weight:900">enclone</span
           <p><a name="software" style="display:block; position:relative; top:-20px;"></a></p>
           <h2>The <span style="color:rgb(120,123,175);font-weight:900">enclone</span> software</h2>
 
-          <p><span style="color:rgb(120,123,175);font-weight:900">enclone</span> is beta software<sup>††</sup> released
+          <p><span style="color:rgb(120,123,175);font-weight:900">enclone</span> was a beta software<sup>††</sup> released
```
Comment (macklin-10x) [inline index.html:242]:
```suggestion
          <p><span style="color:rgb(120,123,175);font-weight:900">enclone</span> was beta software<sup>††</sup> released
```

File: pages/index.html.src [hunk 5]
```diff
@@ -7,8 +7,15 @@
 <img src="img/enclone_banner.png" alt="enclone banner" title="enclone banner" width=100% />
 </a>
 
+<p style="border: 2px; border-style: solid; border-color: black; background-color: #EBF4FC; 
+padding: 8px; font-size: 110%">
+#enclone is deprecated and will no longer receive updates or feature enhancements. 
+#It will remain available for a limited time but will be disabled the future. We recommend transitioning to the improved functionality now integrated directly within Cell Ranger.
```
Comment (macklin-10x) [inline pages/index.html.src:13]:
```suggestion
#It will remain available for a limited time but will be removed in the future. We recommend transitioning to the improved functionality now integrated directly within Cell Ranger version 9.0 and later.
```

File: pages/index.html.src [hunk 6]
```diff
@@ -23,7 +30,7 @@
         acids.</p>
 <p style="border: 2px; border-style: solid; border-color: black; background-color: #EBF4FC; 
 padding: 8px; font-size: 110%">
-#enclone (beta) is provided as a tool for use by the community to accelerate immunology research.  
+#enclone is provided as a tool for use by the community to accelerate immunology research.  
 #enclone is only supported via
 <a href="mailto:enclone@10xgenomics.com">#enclone@10xgenomics.com</a>.
```
Comment (macklin-10x) [inline pages/index.html.src:35]: We should remove the enclone mailing list call-out.

File: yarn.lock [hunk 7]
```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (macklin-10x) [inline yarn.lock:1]: Why is this file here?

File: index.html (hunk 4 above)
Comment (macklin-10x) [reply index.html:242]: This file appears to have been auto-generated from `index.html.src` but the two appear different. Was this file edited by hand?

**Review by evolvedmicrobe — APPROVED** [review]

File: index.html (hunk 4 above)
Comment (abhijna) [reply index.html:239]: @macklin-10x I did a find in folder for the word "beta". Should I just delete this file?

File: pages/index.html.src (hunk 6 above)
Comment (abhijna) [reply pages/index.html.src]: Done! Tried not to touch help text and warning message here though. Could you please double check that I didn't break anything?

File: index.html (hunk 4 above)
Comment (macklin-10x) [reply index.html:239]: Definitely not this one. If anything, we should delete the `.src` versions since we're never going to regenerate the help site.

File: pages/index.html.src (hunk 6 above)
Comment (macklin-10x) [reply pages/index.html.src]: Let's just delete the `.src` files.

**Review by macklin-10x — APPROVED** [review]

File: pages/auto/install_issues.html [hunk 8]
```diff
@@ -68,37 +68,6 @@ <h2>Case 2: download is so slow that it doesn't finish</h2>
 
 <hr>
 
-<h2>Case 3: installation seems to succeed but you get <code>command not found</code></h2>
```
[the remark this answers is not in the document]
Comment (evolvedmicrobe) [reply pages/auto/install_issues.html:71]: Actually never mind, just went ahead and made the changes myself to avoid churn, going to merge.

**Review by evolvedmicrobe — APPROVED** [review]
